### PR TITLE
Edit the Uninstall Teleport guide

### LINKED ISSUE #46955: Include zypper/sles uninstall
## Applies To

https://goteleport.com/docs/admin-guides/management/admin/uninstall-teleport/#linux

## Details

The `zypper` uninstall is not listed.

## How will we know this is resolved?

A person can uninstall from a zypper install of Teleport.

### PR DESCRIPTION
**Pull request by ptgott**
Closes #46955

Include zypper instructions.

### CODE REVIEW

**Review by mmcallister — APPROVED** [review]

**Review by stevenGravy — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

File: docs/pages/admin-guides/management/admin/uninstall-teleport.mdx [hunk 1]
```diff
@@ -132,6 +132,23 @@ Follow the instructions for your Linux distribution:
...
+  $ sudo zypper -y remove teleport-ent
+  ```
+
+  Uninstall the Teleport zypper repo:
+
+  ```code
+  $ sudo zypper removerepo teleport
+  ```
+
+  </Admonition>
```
Comment (taraspos) [inline docs/pages/admin-guides/management/admin/uninstall-teleport.mdx:151]:
```

Error: MDX compilation failed for file "/codebuild/output/src2829177318/src/teleport/docs-website/docs/admin-guides/management/admin/uninstall-teleport.mdx"

154

Cause: Unexpected closing tag `</Admonition>`, expected corresponding closing tag for `<TabItem>` (136:3-136:58)
```

**Review by taraspos — CHANGES_REQUESTED** [review]
@ptgott it seems like docs are failing to build

File: docs/pages/admin-guides/management/admin/uninstall-teleport.mdx (hunk 1 above)
Comment (ptgott) [reply docs/pages/admin-guides/management/admin/uninstall-teleport.mdx]: Thanks! Fixed in 239150028574c38de7e88265b7e681ccb337aa3b

**Comment by ptgott** [comment]
@taraspos Checking whether my changes satisfy your feedback. Thanks!

**Review by taraspos — APPROVED** [review]

**Pull request merged**